pallets/gear/src/benchmarking/tests/syscalls_integrity.rs

```diff
@@ -43,6 +43,7 @@ where
     T::AccountId: Origin,
 {
     SysCallName::all().for_each(|sys_call| {
+        log::trace!("test {sys_call:?}");
```

Comment:
As you discovered recently it is better to use `debug`, isn't it?

Reply:
yes but it's in tests) but ok will make it as debug

Reply:
done